src/libraries/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -687,5 +690,70 @@ private class ClassWithMultipleDictionaries
 
             public Dictionary<string, object> ActualDictionary { get; set; }
         }
+
+        [Fact]
+        public static void CustomObjectConverterInExtensionProperty()
+        {
+            const string Json = "{\"hello\": \"world\"}";
+
+            var options = new JsonSerializerOptions();
+            options.Converters.Add(new JsonObjectConverter());
+
+            ClassWithExtensionPropertyAsObject obj = JsonSerializer.Deserialize<ClassWithExtensionPropertyAsObject>(Json, options);
+            object overflowProp = obj.MyOverflow["hello"];
+            Assert.IsType<string>(overflowProp);
+            Assert.Equal("world!!!", ((string)overflowProp));
+
+            string newJson = JsonSerializer.Serialize(obj, options);
+            Assert.Equal("{\"hello\":\"world!!!\"}", newJson);
+        }
+
+        private class JsonObjectConverter : JsonConverter<object>
+        {
+            public override object Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+            {
+                return reader.GetString() + "!!!";
+            }
+
+            public override void Write(Utf8JsonWriter writer, object value, JsonSerializerOptions options)
+            {
+                // Since we are converter for object, the Hashtable converter will be called instead of this.
```

Comment:
Hashtable? Why would that converter be called? This should say `string converter` since that's the runtime type.

Reply:
Yes thanks. Typo carried over from previous iteration of that test.